phoenix-core/src/it/java/org/apache/phoenix/end2end/ConnectionIT.java

```diff
@@ -58,6 +59,7 @@ public static synchronized void setUp() throws Exception {
         conf = hbaseTestUtil.getConfiguration();
         setUpConfigForMiniCluster(conf);
         conf.set(HConstants.ZOOKEEPER_ZNODE_PARENT, "/hbase-test");
+        conf.set("hbase.client.registry.impl", ZKConnectionInfo.ZK_REGISTRY_NAME);
```

Comment:
We only need to suppor up to 2.4.0, and that one has CLIENT_CONNECTION_REGISTRY_IMPL_CONF_KEY, so better use that.